schema/zone-bundle-metadata.json

```diff
@@ -43,10 +43,10 @@
       "description": "The reason or cause for a zone bundle, i.e., why it was created.",
       "oneOf": [
         {
-          "description": "Generated in response to an explicit request to the sled agent.",
+          "description": "Some other, unspecified reason.",
           "type": "string",
           "enum": [
-            "explicit_request"
+            "other"
```

Comment:
One question and one comment.

Why does this break compat? The variants were reordered, but not changed, so my expectation is that deserializing would work the same.

The comment is that this isn't necessary, and I can undo it if it _does_ break compat. I did it originally so that the derived `PartialOrd` implementation would order things in terms of priority. Then I decided that was too implicit, and wrote the sorting code manually to be clear. So I can undo it without any functional impact.

Reply:
Oh, you're totally right, I saw the enum variant rename from `explicit_request` to `other`, and didn't see the reverse down below.

The comment won't break compat -- or it shouldn't, at least!